### ISSUE TITLE
DTI `step` parameter not documented.

### ISSUE THREAD

**Issue by arokem** [nipy/dipy#901]
Need to document what this does at the TensorModel level.

**Comment by sahmed95** [comment]
Is the step same as the next [function](https://github.com/nipy/dipy/blob/master/dipy/reconst/dti.py#L1184) ?
```python
step : int
    The chunk size as a number of voxels.
```

**Comment by arokem** [comment]
Yes. That's correct.

**Comment by sahmed95** [comment]
On it. There are some pep8 issues too:
```python
dipy/reconst/dti.py:1080:80: E501 line too long (94 > 79 characters)
dipy/reconst/dti.py:1176:1: E303 too many blank lines (3)
dipy/reconst/dti.py:1794:5: E265 block comment should start with '# '
dipy/reconst/dti.py:1797:5: E265 block comment should start with '# '
```

**Comment by sahmed95** [comment]
Fixed : #972
